src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -1042,7 +1042,7 @@ public function ignored() {
 
 // phpcs:set Squiz.Commenting.FunctionComment specialMethods[] __construct,__destruct
 
-/** 
+/**
```

Comment:
Please remove this change. It doesn't belong with this PR and may have been there for a reason (as the sniff is about checking docblock formatting). Same for the fixed file.